solutions/count_character.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+A module for counting occurrences of a character in a string.
+
+Created on 12/1/2025
+@author: Mohamed Altayeb
+Group: ET foundations group 16 (Matrix)
+
+Module functions:
+    - count_character: counts occurrences of a character in a string.
+
+"""
+
+
+def count_character(input_string: str, character: str):
```

Comment:
The function docstring was well-formed but could be enhanced for more clarity and alignment with template requirements.

```suggestion
def count_character(input_string: str, character: str) -> int:
```

Reply:
done 👍